### PR TITLE
feat(incidents): Add status UI

### PR DESCRIPTION
**Pull request by lynnagara**

### CODE REVIEW

File: src/sentry/static/sentry/app/views/organizationIncidents/list/index.jsx [hunk 1]
```diff
@@ -14,7 +14,8 @@ import {PageContent, PageHeader} from 'app/styles/organization';
 import PageHeading from 'app/components/pageHeading';
 import BetaTag from 'app/components/betaTag';
 import space from 'app/styles/space';
-import {getStatus} from '../utils';
+
+import Status from '../status';
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/organizationIncidents/list/index.jsx]: Should this be an absolute import? What kind of guidelines should we have for when to use a relative vs absolute import?

File: src/sentry/static/sentry/app/views/organizationIncidents/list/index.jsx [hunk 2]
```diff
@@ -41,7 +42,9 @@ class OrganizationIncidentsBody extends AsyncComponent {
           <Link to={`/organizations/${orgId}/incidents/${incident.identifier}/`}>
             {incident.title}
           </Link>
-          <div>{getStatus(incident.status)}</div>
+          <div>
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/organizationIncidents/list/index.jsx]: Is this extra `div` needed?

File: src/sentry/static/sentry/app/views/organizationIncidents/status.jsx [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  };
+
+  render() {
+    const isIncidentResolved = isResolved(this.props.incident);
+
+    const icon = isIncidentResolved ? 'icon-circle-check' : 'icon-circle-exclamation';
+    const text = isIncidentResolved ? 'Resolved' : 'Active';
+
+    return (
+      <Container>
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/organizationIncidents/status.jsx]: I think for these smaller, re-used components, we should allow them to be styled using emotion by either passing in `className` to `Container`, or export the styled component instead.

File: src/sentry/static/sentry/app/views/organizationIncidents/list/index.jsx (hunk 1 above)
Comment (lynnagara) [reply src/sentry/static/sentry/app/views/organizationIncidents/list/index.jsx]: I've always been using relative when it's part of that view (e.g. this is part of the shared incident utils)

File: src/sentry/static/sentry/app/views/organizationIncidents/list/index.jsx (hunk 2 above)
Comment (lynnagara) [reply src/sentry/static/sentry/app/views/organizationIncidents/list/index.jsx]: I don't think so

File: src/sentry/static/sentry/app/views/organizationIncidents/status.jsx (hunk 3 above)
Comment (lynnagara) [reply src/sentry/static/sentry/app/views/organizationIncidents/status.jsx]: Do we need views that aren't considered shared components (i.e. in the views folder and not used outside incidents) to implement this behavior even when it's not used? My vote is no. It's likely that we will actually need this here for the subscribe status button, but I'd prefer to add when needed.

Comment (billyvg) [reply src/sentry/static/sentry/app/views/organizationIncidents/status.jsx]: I would say no - views are generally not shared.

**Review by billyvg — APPROVED** [review]

File: src/sentry/static/sentry/app/views/organizationIncidents/status.jsx [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+export default class Status extends React.Component {
+  static propTypes = {
+    incident: SentryTypes.Incident.isRequired,
+  };
+
+  render() {
+    const isIncidentResolved = isResolved(this.props.incident);
+
+    const icon = isIncidentResolved ? 'icon-circle-check' : 'icon-circle-exclamation';
+    const text = isIncidentResolved ? 'Resolved' : 'Active';
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/organizationIncidents/status.jsx]: `t()`?

**Pull request merged**